package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@unvt/charites",
-  "version": "0.5.4",
+  "version": "2.0.0",
```

```suggestion
  "version": "1.0.0",
```

let's follow [semantic versioning](https://semver.org/) and increase a major version since it has breaking changes instead of skipping a version to v2.

---

I will release v1.0.0 after this release.

see: https://github.com/unvt/charites/issues/170#issuecomment-1607336119 this comment.